dll/win32/browseui/resource.h

```diff
@@ -161,22 +161,25 @@
 #define IDB_SHELL_BRANDBAND_MD_LO   246
 #define IDB_SHELL_BRANDBAND_LG_LO   247*/
 
-#define IDD_SEARCH_DLG          1000
-#define IDC_SEARCH_LABEL        1001
-#define IDC_SEARCH_FILENAME     1002
-#define IDC_SEARCH_QUERY        1003
-#define IDC_SEARCH_BUTTON       1004
-#define IDC_SEARCH_STOP_BUTTON  1005
-#define IDC_SEARCH_COMBOBOX     1006
-#define IDC_SEARCH_HIDDEN       1007
-#define IDS_SEARCHINVALID       4518
-#define IDS_COL_NAME            8976
-#define IDS_COL_LOCATION        8977
-#define IDS_COL_RELEVANCE       8989
-#define IDS_SEARCH_FILES_FOUND  9232
-#define IDS_SEARCH_FOLDER       9234
-#define IDS_SEARCH_RESULTS      30520
-#define IDS_SEARCH_OPEN_FOLDER  40960
+#define IDD_SEARCH_DLG           1000
+#define IDC_SEARCH_LABEL         1001
+#define IDC_SEARCH_FILENAME      1002
+#define IDC_SEARCH_QUERY         1003
+#define IDC_SEARCH_BUTTON        1004
+#define IDC_SEARCH_STOP_BUTTON   1005
+#define IDC_SEARCH_COMBOBOX      1006
+#define IDC_SEARCH_HIDDEN        1007
+#define IDS_SEARCHINVALID        4518
+#define IDS_COL_NAME             8976
+#define IDS_COL_LOCATION         8977
+#define IDS_COL_RELEVANCE        8989
+#define IDS_SEARCH_FILES_FOUND   9232
+#define IDS_SEARCH_FOLDER        9234
+#define IDS_SEARCH_RESULTS       30520
+#define IDS_SEARCH_OPEN_FOLDER   40960
+#define IDS_COL_NAME_COLNUM      0
+#define IDS_COL_LOCATION_COLNUM  1
+#define IDS_COL_RELEVANCE_COLNUM 2
```

Comment:
Why does the github web review show so much diff here, while in fact only 3 lines have been added at the end? (as far as I can see at least)

Reply:
I do see now. The padding has been changed without any need. Revert that! Rejected.

Reply:
And COLNUM should be COLUMN by the way, or something shorter.

Reply:
I don't agree with "just use shorter names". Use descriptive names please. If that means the longest name gets longer, so be it.
That said, I think it would be more descriptive not to start these constant names with IDS_ because they are not string resource indexes. And, I don't think they should be grouped with all the other ID* constants, since those are resource identifiers, and these new ones aren't. That solves the alignment issue.

Just COLNUM_NAME or COLUMN_ID_NAME seems fine to me.

Reply:
It's the ID of the column eg (IDS_COL_NAME) appended with COLNUM to create a define which describes the column number for the Column. This is done so that there can be seen a strict dependency between the COLNUM and the column names.

I fail to see how padding is a cause for concern here. The new code is in alignment with the old code, the new definitions are in alignment with the old (as explained above). Even the padding is added to be in alignment with what the file looked like before.

Reply:
I'd be okay with the current names, but they should be grouped separately from the resource identifiers and have a comment explaining why they're called what they are.

Reply:
I do agree to Thomas suggestion to separate them from the rest. Good recommendation!
colNUM is still a really bad choice. Sooner or later somebody will be tempted to rename them to column (erroneously).
I do suggest something like
COLUMN_ID_LOCATION
COLUMN_ID_RELEVANCE
COLUMN_ID_NAME
and separation from the block above

Reply:
Done